test/helpers/increaseTime.js

```diff
@@ -29,7 +29,7 @@ export default function increaseTime(duration) {
 }
 
 // sets time to given timestamp based on current block time
-export function setTimeTo(timestamp) {
+export async function setTimeTo(timestamp) {
```

Comment:
I think I prefer the synchronous approach, has the least opportunity to screw up.